tests/test_core.py

```diff
@@ -9401,6 +9401,34 @@ def test_js_library_i64_params(self):
     self.emcc_args += ['--js-library=' + test_file('core/js_library_i64_params.js')]
     self.do_core_test('js_library_i64_params.c')
 
+  def test_rust_gxx_personality_v0(self):
```

Comment:
Tests in `core` run in all opt modes. For this, which tests a linking issue, I think we can put it in `other` which is more efficient.